tools/webdriver/webdriver/transport.py

```diff
@@ -10,6 +10,72 @@
 """Implements HTTP transport for the WebDriver wire protocol."""
 
 
+missing = object()
+
+
+class ResponseHeaders(dict):
```

Comment:
Shall we inherit from `collections.Mapping` instead? You'd only need to implement `__getitem__`, `__iter__`, `__len__` (and of course `__init__` and `get_list`); the rest will be mixed in for you.

Reply:
Yes, a good idea (done). We can probably drop `get_list` until we have a user, I expect.